com.unity.cinemachine/Runtime/Behaviours/Cinemachine3rdPersonAim.cs

```diff
@@ -77,6 +77,8 @@ private void Reset()
 
         void DrawReticle(CinemachineBrain brain)
         {
+            if (!brain.IsLive(VirtualCamera) || brain.OutputCamera == null)
```

Comment:
Why is this file even being touched?  Did this get removed by mistake in the cm3 branch?